scripts/build/companion_libs/220-ncurses.sh

```diff
@@ -49,7 +49,7 @@ do_ncurses_for_host() {
     # (i.e. in simple cross or native): ncurses may not be needed for
     # host, but we still need them on build to produce 'tic'.
     case "${CT_TOOLCHAIN_TYPE}" in
-        native|cross)   return 0;;
+        native)   return 0;;
```

Comment:
But in case of 'cross' build, library/include paths for host point to the same sysroot as for build. E.g.:
````
CT_CFLAGS_FOR_BUILD='-O2 -g '
CT_CFLAGS_FOR_HOST='-O2 -g -pipe  -I/home/avn/work/ctng/crosstool-ng/.build/sh4-unknown-linux-uclibc/buildtools/include'
CT_LDFLAGS_FOR_BUILD=' '
CT_LDFLAGS_FOR_HOST='  -L/home/avn/work/ctng/crosstool-ng/.build/sh4-unknown-linux-uclibc/buildtools/lib'
````
As you see, it is the same directory where build step places the libraries/headers.